Tools/peg_generator/scripts/show_parse.py

```diff
@@ -84,14 +90,13 @@ def print_parse(source: str, verbose: bool = False) -> None:
 
 def main() -> None:
     args = parser.parse_args()
-    if args.diff and not args.grammar_file:
-        parser.error("-d/--diff requires -g/--grammar-file")
+    new_parser = args.parser == "new"
     if args.multiline:
         sep = "\n"
     else:
         sep = " "
     program = sep.join(args.program)
-    if args.grammar_file:
+    if new_parser:
         tree = _peg_parser.parse_string(program)
 
         if args.diff:
```

Comment:
I'm actually curious if we shouldn't swap `a` and `b` here. Wouldn't it make more sense if `a` was the old parser's tree and `b` was the new parser's tree? That's how we usually view diffs -- `a` (or "left") is the old code, `b` ("right") is the new code.